R/tm_g_bivariate.R

```diff
@@ -565,7 +565,7 @@ srv_g_bivariate <- function(id,
       obj %>%
         teal.code::eval_code(
           c(
```

Comment:
Except here,

Can you break this into 2 `eval_code`?

Why? When concatennating "character" with "expression" (as here) it transforms it in an expression and the code that's run is a "string", not the actual contents.

 I detected this while testing something else.

My suggestion:

```r
      obj %>%
        teal.code::eval_code("library(ggplot2);library(dplyr)") %>% # nolint: quotes)
        teal.code::eval_code(
          c(
            as.expression(anl_merged_input()$expr)
          )
        )
```